.cascade-code/left-nav-refactor/_components/left_nav/left_nav_index_format

```diff
@@ -0,0 +1,164 @@
+## Format used by index pages with the brute-force block that loads the entire file tree.
```

Comment:
This one is just an attempted refactor of the left nav velocity format currently in use, `_cascade/formats/level/LeftNav_Velocity`. Didn't make any changes to accommodate the new block we want to use because it doesn't contain some of the file tree information we need to make it function the same way it currently does.